js/5etools-importer.js

```diff
@@ -249,12 +249,12 @@ function d20plusImporter () {
 		character.save({defaulttoken: (new Date()).getTime()});
 	};
 
-	d20plus.importer.addAction = function (character, name, actionText, index) {
-		if (d20plus.cfg.getOrDefault("import", "tokenactions")) {
+	d20plus.importer.$baseAddAction = function (character, baseAction, name, actionText, prefix, index, expand) {
```

Comment:
remove `$` prefix--`$` is reserved for "this method returns a jQuery object"

Reply:
Is there a particular prefix / naming convention that means "this is a private helper function, don't use it externally"?
Or does this even need that?

Reply:
yeah, underscore `_` is the JavaScript go-to